0020-day-two/stand-ups/readme.md

```diff
@@ -0,0 +1,99 @@
+# Stand-ups
+
+The stand-up is a ceremony that is held each day of the sprint. In this ceremony, each contributor will answer three simple questions. This will repeat until each contributor has answered the three following questions.
+
+1. What did you work on yesterday?
+2. What are you working on today?
+3. Do you have any impediments/blockers?
+
+After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.

```

Comment:
If issues are raised during #3 that need further discussion, those issues are moved to a parking lot to be discussed and resolved after stand-up.

Reply:
Fixed in https://github.com/microsoft/code-with-engineering-playbook/pull/67/commits/b922619a3a27ea380a090d99729f54ca8d191c8d.